public/js/app.js

```diff
@@ -1932,28 +1943,44 @@ __webpack_require__.r(__webpack_exports__);
       });
       this.showEditModal = true;
     },
-    deleteBoard: function deleteBoard(id) {
+    confirmDelete: function confirmDelete(id) {
       var _this6 = this;
 
+      axios.get("/boards/".concat(id), {
+        params: {
+          id: "".concat(id)
+        }
+      }).then(function (response) {
+        _this6.board = response.data;
+      }).catch(function (error) {
+        _this6.$swal("Something is wrong! You can't delete this board");
+      });
+      this.showDeleteModal = true;
+    },
+    deleteBoard: function deleteBoard(id) {
+      var _this7 = this;
```

Comment:
What's the idea behind assigning `this` to `_this7`? I see this in multiple places